manifests/charts/gateway/templates/deployment.yaml

```diff
@@ -105,3 +105,4 @@ spec:
       topologySpreadConstraints:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      terminationGracePeriodSeconds: {{ $.Values.terminationGracePeriodSeconds | default 30 }}
```

Comment:
I think this can simply be done through https://istio.io/latest/docs/setup/additional-setup/customize-installation-helm/?

Reply:
Thanks! I might use it, but still the termination grace period is very important for requests that take more that the default time of 30 seconds on ingress gateway pod termination.
Please reconsider merging this, this will help make our lifes a bit easier :)